qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -233,6 +237,11 @@ def backend_version(self) -> str:
         """Return the version of the backend."""
         return self._backend_version
 
+    @property
+    def schedule_dependency(self) -> rx.PyDiGraph:
+        """Return the schedule dependencies in the calibrations."""
+        return self._schedule_dependency.copy()
```

Comment:
The copy is done here to prevent the end users from modifying the DAG. It is a precautionary measure since they should not be allowed to change the DAG. I would thus leave this as is.